### PR TITLE
user group: add a test case for multi line scenario

### PR DESCRIPTION
**Pull request by pc-m**
At the moment only the main line of a user is a queue member (group/queue) deleting a secondary line from a user should not have an impact on the user's group or queue membership.

Depends-On: https://github.com/wazo-platform/xivo-dao/pull/186

### CODE REVIEW

**Comment by xxbillouxx** [comment]
recheck

File: integration_tests/suite/base/test_user_group.py [hunk 1]
```diff
@@ -100,6 +100,30 @@ def test_dissociate(group1, group2, user, line):
...
+@fixtures.group()
+@fixtures.user()
+@fixtures.line_sip()
+@fixtures.line_sip()
+def test_dissociate_with_multi_line_user(group, user, line_1, line_2):
+    with a.user_line(user, line_1), a.user_line(user, line_2, check=False):
+        with a.group_member_user(group, user):
+            # The group member is the interface of the main line of the user.
+            # Deleting the any secondary line should keep the user in the group
+            # Deleting the main line will remove the user from the group
```
[the remark this answers is not in the document]
Comment (fblackburn1) [reply integration_tests/suite/base/test_user_group.py:112]: IMHO remove user from group when deleting main line is more a design behavior than a goal. Leaving user in a group when deleting line would be probably nice too. So no need to enforce this side effect with a test

**Review by fblackburn1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
recheck

**Comment by pc-m** [comment]
regate

**Pull request merged**